mifosng-android/src/instrumentTest/java/com/mifos/mifosxdroid/login/LoginActivityTest.java

```diff
@@ -0,0 +1,119 @@
+package com.mifos.mifosxdroid.login;
+
+import android.app.Activity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.test.core.app.ActivityScenario;
+import androidx.test.rule.ActivityTestRule;
+
+import com.mifos.mifosxdroid.R;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+
+public class LoginActivityTest {
+
+    private static final String TEST_URL_1 = "demo.mifos.org";
+    private static final String TEST_URL_2 = "www.google.com";
+
+    private ActivityScenario activityScenario;
+    @Rule
+    public ActivityTestRule<LoginActivity> activityTestRule =
+            new ActivityTestRule<>(LoginActivity.class);
+    private Activity loginActivity;
+    private EditText et_username, et_password, et_mifos_domain, et_port, et_tenantIdentifier;

```

Comment:
It would be much better if we follow the `camel case` code `style`  inside the `java classes` for writing Android UI Widgets' variable name like `etUserName`.  As I see, even in the existing test classes this style has not been followed which I don't think is the right way.

Reply:
Ok, I will update it accordingly.

Reply:
Done